src/integrationTest/java/org/opensearch/security/DoNotFailOnForbiddenTests.java

```diff
@@ -434,4 +443,39 @@ public void shouldPerformCatIndices_positive() throws IOException {
         }
     }
 
+    @Test
+    public void checkStatsApi() {
+        // As admin creates 2 documents in different indices, can find both indices in search, cat indice & stats APIs
+        try (final TestRestClient client = cluster.getRestClient(ADMIN_USER.getName(), ADMIN_USER.getPassword())) {
+            final HttpResponse createDoc1 = client.postJson("hi1/_doc?refresh=true", "{\"hi\":\"Hello1\"}");
+            createDoc1.assertStatusCode(SC_CREATED);
+            final HttpResponse createDoc2 = client.postJson("hi2/_doc?refresh=true", "{\"hi\":\"Hello2\"}");
+            createDoc2.assertStatusCode(SC_CREATED);
+
+            final HttpResponse search = client.postJson("hi*/_search", "{}");
+            assertThat("Unexpected document results in search:" + search.getBody(), search.getBody(), containsString("2"));
```

Comment:
Would it be better to assert something more specific here? I am assuming the 2 you want here is coming from the count of documents, but it oculd also be coming from hi2/hello2?

Reply:
That's a good callout to improve the test - this could be replaced with `            assertThat(search.getIntFromJsonBody("/hits/total/value"), equalTo(2));`  However, I think we can live with the current version, and if we expand on these tests that is a good area to invest in